### PR TITLE
add filter and eror msg Pubsub

### PR DESCRIPTION
**Pull request by sandisunandar99**
## OVERVIEW

- fix error pubsub when subscribed by filter phone number without nik

### CODE REVIEW

File: helpers/integration/index.js [hunk 1]
```diff
@@ -6,8 +6,15 @@ const {PUBSUB} = require('../constant')
 
 const findUserCases = async(data) => {
   const user = data.user
+  let filter = {}
+  if (user.nik === "" || user.nik === null || user.nik === undefined){
+    filter = {phone_number: user.phone_number}
+  }else{
+    filter = {nik: user.nik}
+  }
```
Comment (firmanJS) [inline helpers/integration/index.js:14]: kaya nya ini bisa sederhana mang 
`if (user.nik){
    filter = {nik: user.nik}
  }else{
     filter = {phone_number: user.phone_number}
  }`

**Review by firmanJS — APPROVED** [review]

File: helpers/integration/index.js (hunk 1 above)
Comment (sandisunandar99) [reply helpers/integration/index.js:14]: mang @firmanJS kondisi gini ga bisa mengakomodir ketika field nya empty string sama null oy
soalnya kalo ky gini 
`if(user.nik)` itu hanya mengakomodir **_undefined_** aja,,
ini evedent nya 

DeepinScreenshot_select-area_20210324094605
DeepinScreenshot_select-area_20210324094626
DeepinScreenshot_select-area_20210324094643

Comment (firmanJS) [reply helpers/integration/index.js:14]: allrright gas

Comment (firmanJS) [reply helpers/integration/index.js:14]: @khihadysucahyo

**Comment by sandisunandar99** [comment]
@khihadysucahyo sundul,,

**Review by khihadysucahyo — APPROVED** [review]

**Pull request merged**